tests/plugins/test_add_labels.py

```diff
@@ -302,6 +302,157 @@ def test_add_labels_aliases(tmpdir, docker_tasker, caplog,
     if expected_log:
         assert expected_log in caplog.text()
 
+@pytest.mark.parametrize('base_fst, base_snd, df_fst, df_snd, expected, expected_log', [
```

Comment:
How to fix `F811 redefinition of unused 'docker_tasker' from line 23`:
* Put `  # noqa` on that line
* Add a comment 'docker_tasker is a fixture, so its not a redefinition' above